crmsh/bootstrap.py

```diff
@@ -2152,15 +2152,24 @@ def set_cluster_node_ip():
             break
 
 
+def stop_cluster_related_service(stop_list, remote_addr=None):
```

Comment:
As this works now as a generic method, maybe it should be renamed to `stop_services` and receive `service` as argument.

Reply:
yep I agree with naming

Reply:
OK, already change to `stop_services`